content/reference/user/organization/index.md

```diff
@@ -35,7 +35,7 @@ Click on the pen icon to edit the **Organization name**.
 {{< img src="credits-consumption.png" alt="Organization credit informations" >}}
```

Comment:
There is a problem with the pie chart on the `credits-consumption.png` image:

[images/credits-consumption.png](https://github.com/gatling/frontline-cloud-doc/pull/115/files#diff-349af8b4f471bd9a0eb171f426f16be744ef3bdcf33d386536f8319a1531ee9d)

Reply:
thx